app.js

```diff
@@ -2,213 +2,65 @@
-            this.hourlyMin);
-    return customers;
-  },
-  cookiesPurchased: function() {
-    var cookies = Math.round(this.avgPersale * this.randomCustomers());
-    return cookies + ' cookies';
-  },
-  cookieSales: function() {
-    var sales = [];
-    var total = 0;
-    for(var i = 0; i < times.length; i++) {
-      if(i === (times.length - 1)) {
-        sales[i] = total + ' cookies';
-        console.log(times[i] + sales[i]);
-      } else {
-        sales[i] = this.cookiesPurchased();
-        console.log(times[i] + sales[i]);
-        total += parseInt(sales[i]);
-      }
-    } return sales;
-  },
-  render: function() {
-    var ulEl = document.getElementById('seaCenter');
-    var salesToday = this.cookieSales();
-    for (var i = 0; i < salesToday.length; i++) {
-      var liEl = document.createElement('li');
-      liEl.textContent = times[i] + salesToday[i];
-      ulEl.appendChild(liEl);
-    }
-  }
-};
-seaCenter.render();
+function CookieStore(storeName, hourlyMin, hourlyMax,cookieMin, cookieMax, avgPersale){
+  this.store = storeName;
+  this.minHourly = hourlyMin;
+  this.maxHourly = hourlyMax;
+  this.minCookie = cookieMin;
+  this.maxCookie = cookieMax;
+  this.avg = avgPersale;
+  this.trackedSales = [];
+}
 
-var capHill = {
-  hourlyMin: 20,
-  hourlyMax: 38,
-  avgPersale: 2.3,
-  randomCustomers: function () {
-    var customers = Math.floor(Math.random() *
-            (this.hourlyMax - this.hourlyMin + 1) +
-            this.hourlyMin);
-    return customers;
-  },
-  cookiesPurchased: function() {
-    var cookies = Math.round(this.avgPersale * this.randomCustomers());
-    return cookies + ' cookies';
-  },
-  cookieSales: function() {
-    var sales = [];
-    var total = 0;
-    for(var i = 0; i < times.length; i++) {
-      if(i === (times.length - 1)) {
-        sales[i] = total + ' cookies';
-        console.log(times[i] + sales[i]);
-      } else {
-        sales[i] = this.cookiesPurchased();
-        console.log(times[i] + sales[i]);
-        total += parseInt(sales[i]);
-      }
-    } return sales;
-  },
-  render: function() {
-    var ulEl = document.getElementById('capHill');
-    var salesToday = this.cookieSales();
-    for (var i = 0; i < salesToday.length; i++) {
-      var liEl = document.createElement('li');
-      liEl.textContent = times[i] + salesToday[i];
-      ulEl.appendChild(liEl);
-    }
-  }
-};
-capHill.render();
+CookieStore.prototype.render = function () {
+  var trEl = document.createElement('tr');
+  var tdEl = document.createElement('td');
+  tdEl.textContent = this.store;
+  trEl.appendChild(tdEl);
+  for(var i = 0; i < times.length; i++) {
+
+    var customers = Math.floor(Math.random() * (this.maxHourly - this.minHourly + 1) + this.minHourly);
+    customerTotal += customers;
+
+    var cookies = Math.round(this.maxCookie - this.minCookie) + this.minCookie;
+    cookiesTotal += cookies;
+
+    var totalHour = customers * cookies;
```

Comment:
You should be multiplying the number of customers for the hour by the average number of cookies per sale. Currently you are getting far too many cookies.